src/components/Carousel/Carousel.js

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+
+const Carousel = ({ items }) => {
+  const [currentItem, setCurrentItem] = useState(0);
+  const handledClickPrevious = () => {
+    setCurrentItem(currentItem == 0 ? items.length - 1 : currentItem - 1);
+  };
+
+  const handledClickNext = () => {
+    setCurrentItem(currentItem == items.length - 1 ? 0 : currentItem + 1);
+  };
+
+  return (
+    <>
+      <h1>Carousel Section</h1>
+      <article>
+        <section className="carousel_main">
+          <h3>{items[currentItem].data.name}</h3>
+          <img
+            src={items[currentItem].data.main_image.url}
+            alt={items[currentItem].data.main_image.alt}
+          />
+        </section>
```

Comment:
For the items that live inside the Carousel, or any other wrapper element, would be better that each of them was a component itself, that can receive the display items as props, so it could be reused and not dependent to a specific data structure.

Reply:
I created a new component Card